compiler/scopelang/ast.ml

```diff
@@ -40,13 +40,13 @@ let rec locations_used (e : 'm expr) : LocationSet.t =
 
 type 'm rule =
   | ScopeVarDefinition of {
-      var : ScopeVar.t Mark.pos;
+      var : ScopeVar.t * Pos.t list;
```

Comment:
For consistency we _might_ want to call that `(ScopeVar.t, Pos.t list) Mark.ed` ; but to be honest I am not sure how much it improves anything.

Reply:
It looks nicer this way anyhow :)